styles.css

```diff
@@ -0,0 +1,69 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+  margin: 80px 0 45px 0;
+  font-family: system-ui;
+}
+
+.header {
+  background: #a3a3a3;
+  color: #000000;
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  height: 100px;
+  padding: 10px;
+  z-index: 3;
+  text-align: center;
+}
+
+.footer {
+  background: #a3a3a3;
+  color: #000000;
+  height: 80px;
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  z-index: 2;
+  display: flex;
+  text-align: center;
+  justify-content: space-around;
+}
+
+.card {
+  margin: 3rem 10%;
+  border: 5px solid #252629;
+  padding: 3% 10%;
+  scroll-margin-top: 100px;
+  position: relative;
+  z-index: 2;
```

Comment:
What does this `z-index` do here? Since the `footer` and the `header` have a `z-index` as well, I don't see the point right now.